pkg/testkit/testkit.go

```diff
@@ -167,6 +167,22 @@ func (tk *TestKit) MustExecWithContext(ctx context.Context, sql string, args ...
 	}
 }
 
+// SkipIfFailpointDisabled will skip the current test if failpoint is not enabled
+func SkipIfFailpointDisabled(t *testing.T) {
```

Comment:
Why add this function? I think we always enable failpoint in ci env.

Reply:
When I develop, I want to see the `git diff` which is full of changes during `failpoint-enable` and running tests without it makes them fail, so I rather skip them with a message if I have not enabled the failpoints.

Reply:
Function removed.